android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -678,13 +678,21 @@ private Boolean installFile(String file, Boolean override, Boolean executable) {
     		// Check path and create it
     		File installDir = new File(boincWorkingDir);
     		if(!installDir.exists()) {
-    			installDir.mkdir();
-    			installDir.setWritable(true); 
+                if (!installDir.mkdir()) {
+                    if(Logging.ERROR) Log.d(Logging.TAG,"Monitor.installFile(): mkdir() was not successful.");
+                }
+
+                if (!installDir.setWritable(true)) {
+                    if(Logging.ERROR) Log.d(Logging.TAG,"Monitor.setWritable(): mkdir() was not successful.");
+                }
     		}
-    		
+
     		if(target.exists()) {
-    			if(override) target.delete();
-    			else {
+    			if(override) {
```

Comment:
Please review the indentation levels of this block. Since we saw this across various changes: can this be a SPACEs vs. TABs problem?